recipe/meta.yaml

```diff
@@ -20,7 +20,7 @@ requirements:
 
 outputs:
   - name: torchvision
-    script: build-torch.sh  # [not win]
+    script: build-torch.sh   # [not win]
```

Comment:
Ha! Glad I'm not the only one who finds this tendency of the migrator annoying. 😅